src/main/java/org/commcare/modern/database/DatabaseHelper.java

```diff
@@ -76,6 +90,26 @@
 
             set = true;
         }
+        if (inverseFieldNames != null) {
+            for (int i=0; i < inverseFieldNames.length; ++i) {
+                String columnName = TableBuilder.scrubName(inverseFieldNames[i]);
+                if (fields != null) {
+                    if (!fields.contains(columnName)) {
+                        continue;
+                    }
+                }
+                if (set) {
+                    stringBuilder.append(" AND ");
+                }
+
+                stringBuilder.append(columnName);
+                stringBuilder.append("!=?");
```

Comment:
SQL has two inequality operators `<>` and `!=`. I'm not sure if there's a meaningful difference between different engines, is that something you've evaluated and we have a clear sense of?

Reply:
They are equivalent for postgres and sqlite, happy to use either one if we have a convention.